charts/fgt/templates/_configmap-environment.tpl

```diff
@@ -61,9 +42,9 @@ data:
         "system": "any",
         "browser": "any",
         "mode": "dev-secure",
-        "vaultDomain":  "traceability",
-        "didDomain":  "traceability",
-        "domain": "traceability",
+        "vaultDomain":  "{{- .Values.config.vaultDomain }}",
```

Comment:
Better use 
`"vaultDomain": {{ .Values.config.vaultDomain | quote }},` instead of adding double quotes on your own like this:
`"vaultDomain": "{{ .Values.config.vaultDomain }}",`

Reply:
Thanks Torsten!
Changes are implemented 👍